js/apps/admin-ui/src/realm-settings/LocalizationTab.tsx

```diff
@@ -167,21 +174,24 @@ export const LocalizationTab = ({ save, realm }: LocalizationTabProps) => {
       return { result };
     },
     ({ result }) => {
-      const bundles = Object.entries(result).slice(first, first + max + 1);
+      const bundles = Object.entries(result)
+        .map<LocaleSpecificEntry>(([key, value]) => ({ key, value }))
+        .sort((a, b) => a.key.localeCompare(b.key));
```

Comment:
Your change drops the "slice" operation, and due to that the pagination in the UI is now broken: It shows lots of pages, and when I paginate the contents stay the same.
Note while the frontend does pagination, the backend doesn't support it.

Suggested change: first sort, then slice.

Reply:
+1 to @ahus1's suggestion here

Reply:
@ahus1 I replaced the logic to use the [useLocaleSort() hook](https://github.com/keycloak/keycloak/blob/a65af2d25486e323c1e5ef415f65ceb8dbdc7e79/js/apps/admin-ui/src/utils/useLocaleSort.ts#L5) suggested by @jonkoops and added the "slice" operation. From my initial testing it now works as expected.